marimo/_output/formatters/formatters.py

```diff
@@ -84,6 +84,72 @@
 ]
 
 
+def patch_finder(
+    finder: Any,
+    third_party_factories: dict[str, FormatterFactory] | None = None,
+    theme: Theme = "light",
+) -> None:
+    if third_party_factories is None:
+        third_party_factories = THIRD_PARTY_FACTORIES
+    # Note: "Vendored" dependencies may not have a find_spec method.
+    # E.g. `six` bundled with a project.
+    original_find_spec = getattr(finder, "find_spec", None)
+    if original_find_spec is None:
+        return
+
+    # We include `original_find_spec` as a kwarg to force it to be bound
+    # to the new `find_spec` method; this is needed because closures are
+    # late-binding and we're in a for loop ...
```

Comment:
This comment is now stale? No longer in a for loop

Reply:
+1 The work around isn't even needed anymore